### PR TITLE
Adds new test for TT

### PR DESCRIPTION
**Pull request by rkodev**
Adds new test for TT

### CODE REVIEW

File: opensrp-family/src/main/java/org/smartregister/family/domain/FamilyMetadata.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public void updateFamilyOtherMemberRegister(String tableName, int currentLimit,
         this.familyOtherMemberRegister = new FamilyOtherMemberRegister(tableName, currentLimit, showPagination);
     }
 
-    public class FamilyRegister {
+    public static class FamilyRegister {
```
Comment (githengi) [inline opensrp-family/src/main/java/org/smartregister/family/domain/FamilyMetadata.java]: what is the reason for this change

File: opensrp-family/src/test/java/org/smartregister/family/presenter/BaseFamilyRegisterFragmentPresenterTest.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    private FamilyRegisterFragmentContract.View view;
+
+    @Mock
+    private FamilyRegisterFragmentContract.Model model;
+
+    private String viewConfigurationIdentifier = "12345";
+
+    @Before
+    public void setUp() {
+        presenter = Mockito.mock(BaseFamilyRegisterFragmentPresenter.class, Mockito.withSettings()
```
Comment (githengi) [inline opensrp-family/src/test/java/org/smartregister/family/presenter/BaseFamilyRegisterFragmentPresenterTest.java]: the class under test should not be mocked including during initializing

Comment (rkodev) [reply opensrp-family/src/test/java/org/smartregister/family/presenter/BaseFamilyRegisterFragmentPresenterTest.java]: @githengi this mock calls the real methods 
`Mockito.mock(BaseFamilyRegisterFragmentPresenter.class, Mockito.withSettings()
                .useConstructor(view, model, viewConfigurationIdentifier)
                .defaultAnswer(Mockito.CALLS_REAL_METHODS));`

The class `BaseFamilyRegisterFragmentPresenter` is abstract and I needed an instance to test it.

File: opensrp-family/src/main/java/org/smartregister/family/domain/FamilyMetadata.java (hunk 1 above)
Comment (rkodev) [reply opensrp-family/src/main/java/org/smartregister/family/domain/FamilyMetadata.java]: Changing `FamilyRegister` from an inner class to a nested class `FamilyRegister`. 
Should allow creating an instance of FamilyRegister outside the scope of `FamilyMedata` without breaking underlying functionality
